docker/docker-compose.yml

```
@@ -0,0 +1,54 @@
+      - 9000:9000
+      - 9001:9001
+    environment:
+      MINIO_ACCESS_KEY: ${MINIO_ACCESS_KEY}
+      MINIO_SECRET_KEY: ${MINIO_SECRET_KEY}
+    volumes:
+      - ../miniodata:/data
+    command: server /data --console-address ":9001"
+
+
+networks:
+    spotease-network:
```

yamllint

[warning] wrong indentation: expected 2 but found 4

(indentation)